### ISSUE TITLE
pty.node: unknown file type

### ISSUE THREAD

**Issue by l4u** [iraasta/autocomplete-erlang#2]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 0.192.0
**System**: Mac OS X 10.10.3
**Thrown From**: [autocomplete-erlang](https://github.com/iraasta/autocomplete-erlang) package, v0.2.4
### Stack Trace

Failed to load the autocomplete-erlang package

```
At dlopen(/Users/Leo/.atom/packages/autocomplete-erlang/lib/server/pty.js/build/Release/pty.node, 1): no suitable image found.  Did find:
    /Users/Leo/.atom/packages/autocomplete-erlang/lib/server/pty.js/build/Release/pty.node: unknown file type, first eight bytes: 0x7F 0x45 0x4C 0x46 0x02 0x01 0x01 0x00

Error: dlopen(/Users/Leo/.atom/packages/autocomplete-erlang/lib/server/pty.js/build/Release/pty.node, 1): no suitable image found.  Did find:
    /Users/Leo/.atom/packages/autocomplete-erlang/lib/server/pty.js/build/Release/pty.node: unknown file type, first eight bytes: 0x7F 0x45 0x4C 0x46 0x02 0x01 0x01 0x00
    at Error (native)
    at Object.module.(anonymous function) (ATOM_SHELL_ASAR.js:118:20)
    at Object.module.(anonymous function) [as .node] (ATOM_SHELL_ASAR.js:118:20)
    at Module.load (module.js:347:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at Object.<anonymous> (/Users/Leo/.atom/packages/autocomplete-erlang/lib/server/pty.js/lib/pty.js:8:11)
    at Module._compile (module.js:452:26)
    at Object.loadFile [as .js] (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/babel.js:162:21)
```
### Commands

```
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "base16-tomorrow-dark-theme"
    ],
    "disabledPackages": [
      "minimap",
      "jshint",
      "highlight-line",
      "atom-lint",
      "linter-flow"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-color-highlight, v3.0.8
atom-typescript, v2.15.10
autoclose-html, v0.15.0
autocomplete-clang, v0.6.4
autocomplete-css, v0.6.0
autocomplete-elixir, v1.1.2
autocomplete-erlang, v0.2.4
autocomplete-html, v0.5.0
autocomplete-paths, v1.0.2
autocomplete-plus, v2.11.0
autocomplete-snippets, v1.2.0
color-picker, v1.7.0
docblockr, v0.6.3
editorconfig, v0.3.3
file-icons, v1.5.4
git-log, v0.3.0
git-projects, v1.13.0
git-tab-status, v1.7.0
highlight-selected, v0.9.2
javascript-snippets, v1.0.0
jsformat, v0.7.18
language-SCSS, v0.2.0
language-elixir, v0.4.0
language-erlang, v2.0.0
language-haml, v0.19.0
language-jade, v0.4.0
language-swift, v0.4.0
linter, v0.12.0
linter-clang, v2.25.1
linter-coffeelint, v0.2.1
linter-cpplint, v0.1.4
linter-csslint, v0.0.11
linter-elixirc, v0.2.1
linter-erlc, v0.2.0
linter-haml, v0.1.1
linter-hlint, v0.3.1
linter-htmlhint, v0.0.13
linter-js-yaml, v1.0.0
linter-jscs, v1.10.0
linter-jshint, v0.1.2
linter-less, v0.3.1
linter-rubocop, v0.2.2
linter-ruby, v0.1.4
linter-scss-lint, v0.0.14
linter-swiftc, v0.2.2
linter-tslint, v0.2.1
markdown-format, v2.5.0
meteor-api, v2.12.0
permute, v0.1.0
pretty-json, v0.3.3
project-manager, v1.15.5
react, v0.11.4
script, v2.19.0
tabs-to-spaces, v0.9.2
text-manipulation, v0.4.0
travis-ci-status, v0.13.0
vim-mode, v0.44.0
wakatime, v4.0.2

# Dev
No dev packages
```

**Comment by wende** [comment]
Yes. It appears that pseudo-terminal can't get to work on OSX systems.
The issue remains open.
I'll try to find a solution

**Comment by wende** [comment]
Fixed in v1.0
